tests/publish/nitf_formatter_tests.py

```diff
@@ -105,6 +105,62 @@ def test_html2nitf(self):
             </div>""").replace('\n', '').replace(' ', '')
         self.assertEqual(etree.tostring(nitf, encoding='unicode').replace('\n', '').replace(' ', ''), expected)
 
+    def test_html2nitf_br(self):
+        """Check that <br/> is kept if it is a child of and enrichedText parent element"""
+        html = etree.fromstring(dedent("""\
+            <div>
+                <br/>the previous tag should be removed (but not the text)
+                    <p>
+                        the following tag <br/> should still be here
```

Comment:
@petrjasek actually this method is only used by NTB which replaces all <br> by <br/>. I've added the same behaviour in Formatter so that we'll not get surprise if we move core NITF formatter to html2nitf. I keep the <br/>  because that's what html2nitf get.